apis/nucleus/src/components/listbox/ListBoxInline.jsx

```diff
@@ -337,7 +343,7 @@ function ListBoxInline({ options, layout }) {
                   rangeSelect={rangeSelect}
                   checkboxes={checkboxes}
                   height={height}
-                  width={width}
+                  width={width - 3}
```

Comment:
This will reintroduce the problem of having same margin on both sides in grid mode. It has to be added somewhere else.